{"message":"Unable to ensure uniqueness for usernames: ","name":"MongoError","stack":"MongoError: cannot do raw queries on admin in atlas\n    at Function.MongoError.create (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\error.js:31:11)\n    at queryCallback (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\cursor.js:197:34)\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\connection\\pool.js:461:18\n    at _combinedTickCallback (internal/process/next_tick.js:73:7)\n    at process._tickDomainCallback (internal/process/next_tick.js:128:9)","ok":0,"errmsg":"cannot do raw queries on admin in atlas","code":8000,"codeName":"AtlasError","level":"warn","timestamp":"2017-05-28T16:50:28.607Z"}
{"message":"Unable to ensure uniqueness for user email addresses: ","name":"MongoError","stack":"MongoError: cannot do raw queries on admin in atlas\n    at Function.MongoError.create (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\error.js:31:11)\n    at queryCallback (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\cursor.js:197:34)\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\connection\\pool.js:461:18\n    at _combinedTickCallback (internal/process/next_tick.js:73:7)\n    at process._tickDomainCallback (internal/process/next_tick.js:128:9)","ok":0,"errmsg":"cannot do raw queries on admin in atlas","code":8000,"codeName":"AtlasError","level":"warn","timestamp":"2017-05-28T16:50:28.649Z"}
{"message":"Unable to ensure uniqueness for role name: ","name":"MongoError","stack":"MongoError: cannot do raw queries on admin in atlas\n    at Function.MongoError.create (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\error.js:31:11)\n    at queryCallback (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\cursor.js:197:34)\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\connection\\pool.js:461:18\n    at _combinedTickCallback (internal/process/next_tick.js:73:7)\n    at process._tickDomainCallback (internal/process/next_tick.js:128:9)","ok":0,"errmsg":"cannot do raw queries on admin in atlas","code":8000,"codeName":"AtlasError","level":"warn","timestamp":"2017-05-28T16:50:28.650Z"}
{"message":"Unable to ensure uniqueness for usernames: ","name":"MongoError","stack":"MongoError: authentication fail\n    at Function.MongoError.create (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\error.js:31:11)\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\topologies\\replset.js:1195:38\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\connection\\pool.js:750:7\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\connection\\pool.js:726:20\n    at finish (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\auth\\scram.js:163:16)\n    at handleEnd (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\auth\\scram.js:173:7)\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\auth\\scram.js:273:11\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\connection\\pool.js:461:18\n    at _combinedTickCallback (internal/process/next_tick.js:73:7)\n    at process._tickDomainCallback (internal/process/next_tick.js:128:9)","errors":[{"name":"nodeapp-shard-00-00-eutg2.mongodb.net:27017","err":{"name":"MongoError","message":"bad auth Authentication failed.","ok":0,"errmsg":"bad auth Authentication failed.","code":8000,"codeName":"AtlasError"}},{"name":"nodeapp-shard-00-01-eutg2.mongodb.net:27017","err":{"name":"MongoError","message":"bad auth Authentication failed.","ok":0,"errmsg":"bad auth Authentication failed.","code":8000,"codeName":"AtlasError"}}],"level":"warn","timestamp":"2017-05-28T17:10:00.578Z"}
{"message":"Unable to ensure uniqueness for user email addresses: ","name":"MongoError","stack":"MongoError: authentication fail\n    at Function.MongoError.create (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\error.js:31:11)\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\topologies\\replset.js:1195:38\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\connection\\pool.js:750:7\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\connection\\pool.js:726:20\n    at finish (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\auth\\scram.js:163:16)\n    at handleEnd (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\auth\\scram.js:173:7)\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\auth\\scram.js:273:11\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\connection\\pool.js:461:18\n    at _combinedTickCallback (internal/process/next_tick.js:73:7)\n    at process._tickDomainCallback (internal/process/next_tick.js:128:9)","errors":[{"name":"nodeapp-shard-00-00-eutg2.mongodb.net:27017","err":{"name":"MongoError","message":"bad auth Authentication failed.","ok":0,"errmsg":"bad auth Authentication failed.","code":8000,"codeName":"AtlasError"}},{"name":"nodeapp-shard-00-01-eutg2.mongodb.net:27017","err":{"name":"MongoError","message":"bad auth Authentication failed.","ok":0,"errmsg":"bad auth Authentication failed.","code":8000,"codeName":"AtlasError"}}],"level":"warn","timestamp":"2017-05-28T17:10:00.578Z"}
{"message":"Unable to ensure uniqueness for role name: ","name":"MongoError","stack":"MongoError: authentication fail\n    at Function.MongoError.create (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\error.js:31:11)\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\topologies\\replset.js:1195:38\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\connection\\pool.js:750:7\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\connection\\pool.js:726:20\n    at finish (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\auth\\scram.js:163:16)\n    at handleEnd (C:\\nodeapp\\node_modules\\mongodb-core\\lib\\auth\\scram.js:173:7)\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\auth\\scram.js:273:11\n    at C:\\nodeapp\\node_modules\\mongodb-core\\lib\\connection\\pool.js:461:18\n    at _combinedTickCallback (internal/process/next_tick.js:73:7)\n    at process._tickDomainCallback (internal/process/next_tick.js:128:9)","errors":[{"name":"nodeapp-shard-00-00-eutg2.mongodb.net:27017","err":{"name":"MongoError","message":"bad auth Authentication failed.","ok":0,"errmsg":"bad auth Authentication failed.","code":8000,"codeName":"AtlasError"}},{"name":"nodeapp-shard-00-01-eutg2.mongodb.net:27017","err":{"name":"MongoError","message":"bad auth Authentication failed.","ok":0,"errmsg":"bad auth Authentication failed.","code":8000,"codeName":"AtlasError"}}],"level":"warn","timestamp":"2017-05-28T17:10:00.578Z"}
